bqn-comint.el

```diff
@@ -118,7 +118,9 @@ When DONT-FOLLOW is non-nil, maintain focus on the buffer where the function was
         (buffer (current-buffer)))
     (pop-to-buffer (process-buffer session))
     (goto-char (point-max))
-    (insert (format "\n%s\n" region))
+    (insert (format ")escaped \"%s\""
```

Comment:
Reword this commit as below:

> Enables buffer and region evals.

> Following the recommendations from https://github.com/dzaima/CBQN/issues/29

As a rule of thumb, the title should be a short, imperative description of the change you want to implement. The body of commit message is filled with the comments you deem necessary.